test/controllers/event_controller_test.exs

```diff
@@ -0,0 +1,20 @@
+defmodule Dwylbot.EventTestController do
+  use Dwylbot.ConnCase
+  alias Poison.Parser, as: PP
+
+  test "POST /event/new - no error", %{conn: conn} do
+    payload = "./test/fixtures/add_label.json"
+    |> File.read!()
+    |> PP.parse!()
```

Comment:
@shouston3 is the omission of parens for function invocation documented somewhere in one of our repos?